fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/BulkExportJobExecutionResponse.java

```diff
@@ -623,6 +575,12 @@ public static String generate(BulkExportJobExecutionResponse obj, boolean withSe
                         }
                     }
 
+                    if (withSensitive) {
```

Comment:
is this really sensitive?

Reply:
yes, the randomly generated prefix is used to protect the publicaccess exported file in ACL protected COS bucket.   Any user could download the file if he/she knows the prefix and then compose the full path.